lib/src/HttpUtils.cc

```diff
@@ -632,36 +481,56 @@ ContentType getContentType(const std::string &fileName)
     }
 }
 
+// Taken and modified from https://stackoverflow.com/a/8317622/3687637
```

Comment:
I've made a hash implementation in the string_view.h for boost::string_view, why does it not work?

Reply:
Good question. I'll try to figure it out.

Reply:
Solved by following LLVM's solution. Use cityhash64 on 64bit system